test_suite/generic/helpers.py

```diff
@@ -10,3 +11,14 @@ def __get_instance_data_from_json(key_to_find, values_to_find, path=INSTANCES_JS
     for instance in instances_json_data.values():
         if key_to_find in instance.keys() and instance[key_to_find] in values_to_find:
             return instance
+
+
+def check_avc_denials(host):
+    command_to_run = 'x=$(ausearch -m avc 2>&1 &); echo $x'
+    result = test_lib.print_host_command_output(host,
+                                                command_to_run,
+                                                capture_result=True)
+
+    no_avc_denials_found = 'no matches' in result.stdout
+
+    assert no_avc_denials_found, 'There should not be any avc denials (selinux)'
```

Comment:
ATM AVC are not collected in some cases in the `test_no_avc_denials`
IMHO it is allways useful to log AVC; even know ones which should be skipped.

One would be able to check whether previous AVCs are already fixed or whether new one are introduced.

so maybe it would be nice to have a feature to skip tests here (disabled by default)
e.g.
```
def check_avc_denials(host, skip_message=None):

//snip
if skip_message:
    pytest.skip(skip_message)
else:
    assert no_avc_denials_found, 'There should not be any avc denials (selinux)'
```

Reply:
will create another PR for this change.